content/docs/how-we-work/git/git-rollback.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Rolling Back Changes
+weight: 7
+description: Working with forks in GitHub allows us to create a copy of an existing repository and make changes to it without affecting the original project. We can use forks to experiment with new features, fix bugs, or contribute to open source projects.
```

Comment:
@Yemaneberhan-Lemma, mistake in copy/paste?

Reply:
Sorry. Fixed.